pkg/services/sqlstore/alert_notification_test.go

```diff
@@ -321,5 +321,54 @@ func TestAlertNotificationSQLAccess(t *testing.T) {
 				So(len(query.Result), ShouldEqual, 4)
 			})
 		})
+
+		Convey("Notification Uid by Id Caching", func() {
+			ss.CacheService.Flush()
```

Comment:
@papagian I saw that some tests initialize a separate TestDb for isolated test scopes. Perhaps I should use this technique instead? I like it because it's less verbose. https://github.com/grafana/grafana/blob/5aeb367322bbe33d4b95d84c33a05da17676feac/pkg/services/sqlstore/user_test.go#L313-L315

Reply:
Then there's not even a need to flush :)

Reply:
yes, you can do that instead

Reply:
Not that important but my last impression was that you would use a separate test database for these tests and you would remove flushing.